plugins/content_distribution/providers/youtube_api/lib/YoutubeApiDistributionEngine.php

```diff
@@ -41,6 +41,10 @@ class YoutubeApiDistributionEngine extends DistributionEngine implements
 
 	const TIME_TO_WAIT_FOR_YOUTUBE_TRANSCODING = 5;
 
+	const SIZE_2_GB = 2147483648;
+
+	const LONG_PROCESS_TIMEOUT = 1200;
```

Comment:
I think it better to change it to a configuration value instead of const in the code

Reply:
changed to variables